### DISCUSSION TITLE
One GitLab issue which keeps track of all MRs

### DISCUSSION THREAD

**Discussion by chainmywallet** [renovatebot/renovate#31540] [Suggest an Idea]
### Tell us more.

Currently the `dependencyDashboard` is generating one Issue for each repository which keep track of the related updates. We need one unified GitLab Issue which is keeping track of all MRs, preferably over a specified time range. Is there a feature which is expected in the future, or is there already an out of the box solution for this in renovate? 
If not, what is the best workable workaround to achieve this?

**Comment by rarkins** [comment]
There is no feature existing or planned for the future. I don't think such a thing is achievable with a "workaround" either - you would need to build quite a complicated _solution_ of your own.
